src/main/java/com/cloudbees/jenkins/GitHubCommitNotifier.java

```diff
@@ -88,18 +89,20 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
             if (buildResult.equals(Result.FAILURE)) {
                 throw error;
             } else {
-                listener.error("[GitHub Commit Notifier] - " + error.getMessage());
+                listener.error(format("[GitHub Commit Notifier] - %s", error.getMessage()));
                 if (buildResult.isWorseThan(build.getResult())) {
-                    listener.getLogger().println("[GitHub Commit Notifier] - Build result will be set to " + buildResult);
+                    listener.getLogger()
+                            .println(format("[GitHub Commit Notifier] - Build result will be set to %s", buildResult));
```

Comment:
think such line breaks is normal. In future - i'll replace it with localized method call and it will be inlined